### ISSUE TITLE
Accessibility: Filter unable to close widget after opening

### ISSUE THREAD

**Issue by jn394** [microsoft-search/pnp-modern-search#3900]
**Version used**
4

**Describe the bug**
Keyboard navigation of filter widgets. After going to desired widget, pressing the "Down Arrow" key moves the user into the filter options. But the user is no longer able to go back to the widget header and close the widget once opened, by using the arrow keys. Press "Tab" will move the focus to the next widget as well as "Shift + Tab" will move focus back to the previous widget. While using either method to go back to the opened widget header, the focus moves into the filter options.

**To Reproduce**
Detailed steps to reproduce the behavior:
1. Tab to filters
2. Press "Enter" to open widget
3. Press "Down Arrow" to go into options
4. Neither pressing "Up Arrow" will not bring focus back up to Widget header. Nor pressing "Down Arrow" will loop back up to Widget header.
5. Pressing "Tab" or "Shift + Tab" will move focus away from opened widget. But using pressing "Tab" or "Shift + Tab" to go back to the open widget, will open move focus back into the opened widgets filter options

**Expected behavior**
There should be a way for keyboard users to be able to close the opened widget. Once widget is opened the user should be able to "Tab" back to the header of the opened widget to close. 

**Desktop (please complete the following information):**
 - Chrome

**Comment by wobba** [comment]
Fixed in #4805 (commit(s) `a06aa9f4`), now merged into the `develop` branch (merge commit `578dbb3`). The fix will ship in the next release; until then it can be validated by building from `develop`. Closing as resolved — please reopen if the issue persists after testing on `develop`.
